src/lib/viewers/box3d/model3d/__tests__/Model3DControlsNew-test.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { shallow, ShallowWrapper } from 'enzyme';
-import AnimationControls from '../../../controls/model3d/AnimationControls';
+import AnimationControls from '../../../controls/box3d/model3d/AnimationControls';
 import FullscreenToggle from '../../../controls/fullscreen';
 import Model3DControls, { Props } from '../Model3DControlsNew';
-import Model3DSettings, { CameraProjection, RenderMode } from '../../../controls/model3d/Model3DSettings';
-import ResetControl from '../../../controls/model3d/ResetControl';
-import VrToggleControl from '../../../controls/model3d/VrToggleControl';
+import Model3DSettings, { CameraProjection, RenderMode } from '../../../controls/box3d/model3d/Model3DSettings';
+import ResetControl from '../../../controls/box3d/model3d/ResetControl';
```

Comment:
Do we need the `model3d` subdirectory or can everything live in the `box3d` directory?

Reply:
You're right I don't think the subdirectories are needed